maven-scm-providers/maven-scm-providers-git/maven-scm-provider-gitexe/src/test/java/org/apache/maven/scm/provider/git/gitexe/command/status/GitStatusConsumerTest.java

```diff
@@ -52,9 +52,7 @@
     {
         GitStatusConsumer consumer = new GitStatusConsumer( new DefaultLog(), null, relativeRepoPath );
 
-        BufferedReader r = new BufferedReader( new FileReader( gitlog ) );
-
-        try
+        try ( BufferedReader r = new BufferedReader( new FileReader( gitlog ) ) )
```

Comment:
We should create another issue here because it is subject to character conversion.

Reply:
good point. fixed.